src/assets/js/events.js

```diff
@@ -39,7 +39,7 @@ window.addEventListener('DOMContentLoaded', () => {
         e.srcElement.classList.toggle("fa-video-slash");
         
     })
-
+/*
```

Comment:
Lets remove this to avoid more complications and dead code :)

Reply:
done :)